### PR TITLE
feature/microsoft_365_defender_add_verdict_fields

### PR DESCRIPTION
**Pull request by clement-burtscher-sekoia**
## Description

Fixes https://github.com/SekoiaLab/integration/issues/1782

### Added

- Parse custom fields:
    - `microsoft.defender.threat.last_verdict`
    - `microsoft.defender.threat.verdict`

## Type of Change

- [ ] New intake format
- [X] Update to existing intake format
- [ ] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product: Microsoft / Microsoft 365 Defender

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Add verdict-related threat fields to the Microsoft 365 Defender intake format and update supporting metadata and tests.

New Features:
- Capture the final consolidated threat verdict in microsoft.defender.threat.last_verdict.
- Capture the latest threat verdict from ThreatAnalysisSummary in microsoft.defender.threat.verdict.

Enhancements:
- Document the new verdict fields and release date in the Microsoft 365 Defender changelog.
- Extend field metadata and smart descriptions to cover the new verdict fields.

Tests:
- Update existing alert evidence tests and add a new mail cluster verdict test case to validate parsing of the new verdict fields.

### CODE REVIEW

File: Microsoft/microsoft-365-defender/tests/test_alert_evidence.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
         "threat": {
           "detection_status": "Detected",
           "last_remediation_state": "Active",
+          "last_verdict": "Suspicious",
           "remediation_providers": [
             {
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Microsoft/microsoft-365-defender/tests/test_alert_evidence.json:51]: Irrelevant, not related to the addition of these custom fields

File: Microsoft/microsoft-365-defender/tests/test_alert_evidence_5.json [hunk 2]
```diff
@@ -56,6 +56,7 @@
             ]
           },
           "last_remediation_state": "Active",
+          "last_verdict": "Suspicious",
           "remediation_providers": [
             {
               "RemediationDate": "2026-01-13T09:40:15.4348077Z",
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Microsoft/microsoft-365-defender/tests/test_alert_evidence_5.json:62]: Irrelevant, not related to the addition of these custom fields

File: Microsoft/microsoft-365-defender/CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [Unreleased] - 2026-07-15
+
```
Comment (squioc) [inline Microsoft/microsoft-365-defender/CHANGELOG.md]:
```suggestion
## [Unreleased] 

 ## 2026-07-15 - 1.0.6

```

**Review by squioc — CHANGES_REQUESTED** [review]
Hello @clement-burtscher-sekoia,

Thank you for the work.

Some changes to apply.

File: Microsoft/microsoft-365-defender/CHANGELOG.md (hunk 3 above)
Comment (clement-burtscher-sekoia) [reply Microsoft/microsoft-365-defender/CHANGELOG.md]: @squioc fixed with commit [Fix changelog section](https://github.com/SEKOIA-IO/intake-formats/pull/2702/changes/8bba5b32e5c983cd70600e63d1b2a0e3365e946b)

**Review by squioc — APPROVED** [review]
LGTM. Thank you
